src/domain/_common/components/FloatingButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { PlusIcon } from '.';
+
+interface FloatingButtonProps {
+  onClick?: () => void;
+  disabled?: boolean;
+}
+
+export const FloatingButton = ({ onClick = () => {}, disabled = false }: FloatingButtonProps) => {
+  const handleButtonClick = () => {
+    onClick();
+  };
+
+  return (
+    <button
+      onClick={handleButtonClick}
+      disabled={disabled}
+      className="flex h-[52px] w-[52px] items-center justify-center rounded-full bg-highlight-gradient active:bg-highlight-gradient-dark disabled:bg-grey-whitegray disabled:bg-none"
+    >
+      <PlusIcon color={disabled ? 'stroke-grey-placeholder' : 'stroke-white'} />
```

Comment:
svg는 assets에서 다른 .svg 파일들과 함께 저장해서 관리하고,
path에 fill="current" 속성을 주면 color를 부모로부터 상속받아서 동적으로 변경할 수 있게 하는게 어때요?
이렇게하면 svg를 `assets/` 한 곳에서 관리할 수 있고,
매번 svg 컴포넌트를 만드는 수고를 안해도 되어서 좋은 것 같아요~

```suggestion
import { ReactComponent as PlusIcon } from '@/assets/plus.svg';

interface FloatingButtonProps {
  onClick?: () => void;
  disabled?: boolean;
}

export const FloatingButton = ({ onClick = () => {}, disabled = false }: FloatingButtonProps) => {
  const handleButtonClick = () => {
    onClick();
  };

  return (
    <button
      onClick={handleButtonClick}
      disabled={disabled}
      className="flex h-[52px] w-[52px] items-center justify-center rounded-full bg-highlight-gradient stroke-white active:bg-highlight-gradient-dark disabled:bg-grey-whitegray disabled:bg-none disabled:stroke-grey-placeholder"
    >
      <PlusIcon />
```

Reply:
`assets/plus.svg`

```svg
<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
    <path d="M12 5V19" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" fill="current"/>
    <path d="M5 12H19" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" fill="current" />
</svg>
```

Reply:
오! 굉장히 좋은 방법..!!